lightrag/tests/test_litellm_class.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from unittest.mock import Mock
+from lightrag.core.types import ModelType
+from lightrag.components.model_client.lite_client import LiteClient
+
+
+from lightrag.utils import setup_env  # ensure you have .env with OPENAI_API_KEY
+
+setup_env("C:/Users/jean\Documents/molo/LightRAG/.env")  # need to setup env

```

Comment:
we need to mock data in the testing as we did not set up api key this needs to change

Reply:
ok

Reply:
@jeaneigsi can you take this one, the mock data in testing. I've made some changes based on your code for the other parts.

Reply:
yes i delete it